### PR TITLE
Use custom version of Tachyons

### PR DESCRIPTION
**Pull request by sfrdmn**
Resolves cssconf/2018.cssconf.eu#50

### CODE REVIEW

File: contents/css/_vendor.tachyons.css [hunk 1]
```diff
@@ -1994,7 +1995,7 @@ code, .code { font-family: Consolas, monaco, monospace; }
 .nested-headline-line-height h5, .nested-headline-line-height h6 { line-height: 1.25; }
 .nested-list-reset ul, .nested-list-reset ol { padding-left: 0; margin-left: 0; list-style-type: none; }
 .nested-copy-indent p+p { text-indent: 1em; margin-top: 0; margin-bottom: 0; }
-.nested-copy-seperator p+p { margin-top: 1.5em; }
+.nested-copy-separator p+p { margin-top: 1.5em; }
```
Comment (Kriesse) [inline contents/css/_vendor.tachyons.css]: nice

File: contents/css/_vendor.tachyons.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*! TACHYONS v4.9.0 | http://tachyons.io */
+/*! TACHYONS v1.0.0-wwwtf2018 | http://tachyons.io */
```
Comment (Kriesse) [inline contents/css/_vendor.tachyons.css]: can we have two separate forks for each conf? unfortunately the layout systems are quite different and we won't be able to re-use it.

**Comment by lukaszklis** [comment]
@sfrdmn you should have access to do it now (we’ve already forked the tachyons repo). Feel free to ping me in case it still doesn’t allow you to do a similar PR to JSConf’s version.

**Comment by sfrdmn** [comment]
@Kriesse @lukaszklis Alright I changed it so it now points to the custom repo as an NPM dep

One problem: the download is annoyingly slow as the Tachyons repo is pretty big for some reason (60 meg). Will suck for deploy. Could revert back to including the build artifact here or actually just publish it to NPM (eg `@cssconf/tachyons-cssconf-2018`)

**Comment by sfrdmn** [comment]
Will do JSConf with whatever pattern we establish here

**Comment by sfrdmn** [comment]
@lukaszklis any input on this one?

**Comment by lukaszklis** [comment]
@sfrdmn have you tried using https://github.com/tachyons-css/tachyons-generator yet? I wanted to look into right after the speaker pages are ready

**Comment by sfrdmn** [comment]
No, not familiar with that. Can check it out

**Pull request closed without merging**